src/helper/log/Log.js

```diff
@@ -7,14 +7,19 @@ import Base from '../Base.js';
 class Log extends Base {
   /**
    *
-   * @param {import('../../client/WOLF.js').default} client
+   * @param {WOLF} client
```

Comment:
This wont actually work, I don't think?
Should still be importing as it was

Reply:
Done, if WOLF.js used "@/helper/Somthin.js" it's better than "../../".
And, if used, Class import rather than models.WolfError() Is being easily added to the JSDoc.